net-analyzer/wireshark/files/4.4.6-return-from-main.patch

```diff
@@ -0,0 +1,59 @@
+Patch from: https://gitlab.com/wireshark/wireshark/-/commit/b4d6d76c6eba2da7f8df55c328607fe651037c03
```

Comment:
As a nit for future, I like to include Gentoo bug references (and any others I can find) at the top of patches as well. It means the patch lives standalone in case other distros pick it up etc.